### PR TITLE
[7-14]Feature coach check user test

### PR DESCRIPTION
**Pull request by KotdenGit**
1.Added display of the selected test from the list.
2.Modules for checking and displaying information are connected.

### CODE REVIEW

File: exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts [hunk 1]
```diff
@@ -8,10 +11,73 @@ import { Router } from '@angular/router';
...
+	constructor(private router: Router, private apiService: ApiService) {
+		this.passedTestId = 1;
+		//this.passedTestId = this?.router?.getCurrentNavigation()?.extras?.state?.data.id;
+	}
+
+	ngOnInit() {
+		this.apiService
+			.getRequest(API_PATH.GET_CHECK_TEST + `${this.passedTestId}`)
+			.then((data) => {
+				console.log(data.userAnswers);
```
Comment (KuznetsovR) [inline exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts]: Remove console

File: exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts [hunk 2]
```diff
@@ -8,10 +11,73 @@ import { Router } from '@angular/router';
...
+		this.testAuditionAnswers.push(item);
+		delete item.question;
 	}
 
-	ngOnInit(): void {}
+	getCheckQuestionTopic(item: any): void {
+		item.answers = [];
+		this.testTopicAnswers.push(item);
+		delete item.question;
+		console.log(this.testTopicAnswers);
```
Comment (KuznetsovR) [inline exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts]: remove console

File: exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts [hunk 3]
```diff
@@ -8,10 +11,73 @@ import { Router } from '@angular/router';
...
 
-	constructor(private router: Router) {
-		this.passedTestId = this?.router?.getCurrentNavigation()?.extras?.state?.data.id;
+	public testGrammarAnswers: CheckCoachQuestion[] = [];
+	public testAuditionAnswers: CheckCoachQuestion[] = [];
+	public testTopicAnswers: CheckCoachQuestion[] = [];
+
+	constructor(private router: Router, private apiService: ApiService) {
+		this.passedTestId = 1;
+		//this.passedTestId = this?.router?.getCurrentNavigation()?.extras?.state?.data.id;
```
Comment (KuznetsovR) [inline exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts]: remove comment

File: exalearn-ui/src/app/components/check-nav-btn/check-nav-btn.component.ts [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 export class CheckNavBtnComponent implements OnInit {
-	@Input() question: CheckQuestion;
+	@Input() answer: CheckCoachQuestion;
 	@Input() index: number;
 	@Input() currentIndex: number;
 	@Input() questionId: number;
 
 	ngOnInit(): void {
-		this.question.index = this.index;
+		// this.question.index = this.index;
```
Comment (KuznetsovR) [inline exalearn-ui/src/app/components/check-nav-btn/check-nav-btn.component.ts]: remove comment

File: exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts (hunk 3 above)
Comment (KotdenGit) [reply exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts]: Done.

File: exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts (hunk 1 above)
Comment (KotdenGit) [reply exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts]: Done

File: exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts (hunk 2 above)
Comment (KotdenGit) [reply exalearn-ui/src/app/pages/check-test-item-page/check-test-item-page.component.ts]: Done

File: exalearn-ui/src/app/components/check-nav-btn/check-nav-btn.component.ts (hunk 4 above)
Comment (KotdenGit) [reply exalearn-ui/src/app/components/check-nav-btn/check-nav-btn.component.ts]: Ok

**Review by KuznetsovR — APPROVED** [review]

**Review by vstrabko — APPROVED** [review]

**Pull request merged**